src/app/DiagramCanvas.tsx

```diff
@@ -7,6 +7,8 @@ import { getWindowSize } from "../core/Util";
 import { useBetterMemo } from "../core/Hook";
 import React from "react";
 import Konva from "konva";
+// import TestMarkdown from "./TestMarkdown.mdx";
```

Comment:
Please remember to remove redundant test codes for Line 10-11 and 41

Reply:
fixed